.gitignore

```diff
@@ -9,3 +9,8 @@ yarn-debug.log*
 yarn-error.log*
 *.log
 .history
+
+# scss
+maquetado/scss/out/*
```

Comment:
Like I said, it's easier if you set your `css` files inside your `maquetado` folder. With that in mind you just have to do this:
```
maquetado/*.css.map
maquetado/*.sass.map
maquetado/*.scss.map
maquetado/*.css
```